packages/wonder-blocks-modal/components/modal-launcher.js

```diff
@@ -107,11 +111,14 @@ export default class ModalLauncher extends React.Component<Props, State> {
      */
     lastElementFocusedOutsideModal: ?HTMLElement;
 
-    static defaultProps = {
+    static defaultProps: DefaultProps = {
         backdropDismissEnabled: true,
     };
 
-    static getDerivedStateFromProps(props: Props, state: State) {
+    static getDerivedStateFromProps(
+        props: Props,
+        state: State,
+    ): ?Partial<State> {
```

Comment:
I like this type here because I always forget if I have to return the full object for `state` or not.

Reply:
Yeah, I originally was typing "just enough" to get it to pass which was `State` in some cases, `Partial<State>` in others, etc. I updated them all to be `?Partial<State>` to reflect the full range of possibilities.